service/storage_service.cc

```diff
@@ -187,12 +187,27 @@ std::optional<gms::inet_address> storage_service::get_replace_address() {
 }
 
 bool storage_service::is_replacing() {
```

Comment:
Re commit message (`storage_service: is_replacing: rely directly on config options
`): "before we change the latter..." -> "before we remove the latter".

Reply:
yup, will edit it.